content-repo/test_data/bad-mdx-readme.md

```diff
@@ -604,18 +605,18 @@ This integration was integrated and tested with version xx of Proofpoint TAP v2
         }
     ],
     "Proofpoint.MessagesBlocked": [
-        {
+        \{
             "GUID": "2222",
             "QID": "r2FNwRHF004109",
             "ccAddresses": [
                 "bruce.wayne@university-of-education.zz"
             ],
             "clusterId": "pharmtech_hosted",
             "fromAddress": "badguy@evil.zz",
-            "headerCC": "\"Bruce Wayne\" <bruce.wayne@university-of-education.zz>",
-            "headerFrom": "\"A. Badguy\" <badguy@evil.zz>",
+            "headerCC": "[\"Bruce Wayne\" bruce.wayne@university-of-education.zz](mailto:bruce.wayne@university-of-education.zz)",
```

Comment:
[Here](https://docusaurus.io/blog/preparing-your-site-for-docusaurus-v3#bad-usage-of-gfm-autolink) and everywhere